airflow/example_dags/tutorial.py

```diff
@@ -89,18 +89,27 @@
     # [END basic_task]
 
     # [START documentation]
-    dag.doc_md = __doc__
+    with DAG('tutorial',
+        default_args=default_args,
+        schedule_interval=timedelta(days=1)
+    ) as dag:
```

Comment:
This isn't right !

We are already in the DAG context manager, why create it again ?

Reply:
Because of this: https://github.com/apache/airflow/pull/14853#issuecomment-801212303

If you go directly to the section, you see this
![Captura de pantalla 2021-03-18 a las 12 57 50](https://user-images.githubusercontent.com/11339132/111622488-8d23de80-87e9-11eb-95c0-a7ee6a34bbb2.png)

And I think it would be more useful to have those lines in the context of an entire DAG. So if the tutorial dag must run fine, I think it would be better to include the code as text or in some other file

Reply:
The entire file is already shown at the top of the doc though: 

https://airflow.apache.org/docs/apache-airflow/stable/tutorial.html#example-pipeline-definition

Reply:
And if you click on "View Source" on the blue strip it should show you full file too

Reply:
The purpose of showing only few lines is too highlight the user what the section is about